pages/integrations/github.md.erb

```diff
@@ -1,79 +1,83 @@
+1. Open Buildkite’s _Personal Settings_.
+2. Select _[Connected Apps](https://buildkite.com/user/connected-apps)_.
+3. Select the GitHub _Connect_ button: 
+    <%= image "personal-settings.png", width: 1650/2, height: 642/2, alt: "Screenshot of the Buildkite Connected Apps screen" %>
+4. Select _Authorize buildkite_. GitHub redirects you back to your _Connected Apps_ page. 
 
-<%= image "personal-settings.png", width: 1650/2, height: 642/2, alt: "Screenshot of the Buildkite Connected Apps screen" %>
+You can now [set up a pipeline](#set-up-a-new-pipeline-for-a-github-repository).
 
-You'll be prompted to give permission for Buildkite to post status
-updates, and then redirected back to your Connected Apps page. After that you can [set up a pipeline](#setting-up-a-new-pipeline-for-a-github-repository).
+## Set up a new pipeline for a GitHub repository
 
-## Setting up a new pipeline for a GitHub repository
 
-1. Create a new pipeline on Buildkite, using your GitHub repository URL in the form `git@github.com:your/repo`.
+1. Select _Pipelines_ > _New Pipeline_ <svg viewBox="0 0 22 22" width="22px" height="22px" style="fill: currentcolor; vertical-align: middle;border-radius:50%;"><title>New Pipeline</title><rect x="4" y="10" width="14" height="2"></rect>
+<rect x="10" y="4" width="2" height="14"></rect>
+</svg>
+2. Enter your pipeline details, including your GitHub repository URL in the form `git@github.com:your/repo`.
 
     <%= image "new-pipeline.png", width: 1550/2, height: 846/2, alt: "Screenshot of adding a new pipeline " %>
 
-2. Add at least one step to your pipeline, then click "Next". Change these steps at any time in Pipeline Settings.
-
-3. Follow the instructions on the page to set up a webhook:
+3. If you are still using the web steps visual editor, add at least one step to your pipeline. Refer to [Defining Steps - Adding steps](/docs/pipelines/defining_steps#adding-steps) for more information.
+4. Select _Create Pipeline_.
+4. Follow the onscreen instructions to set up a webhook:
 
     1. Add a new webhook in GitHub.
     2. Paste in the provided webhook URL.
     3. Select which events trigger the webhook.
+5. If using the YAML steps editor, add at least one step to your pipeline, then select _Save and Build_. Refer to [Defining Steps - Adding steps](/docs/pipelines/defining_steps#adding-steps) for more information.
 
 If you need to set up the webhook again, you can find the instructions linked at the bottom of the pipeline GitHub settings page.
 
-## Branch configuration and Settings
-
-You can edit the GitHub settings for each pipeline from the pipeline’s settings page:
+ You can edit your pipeline configuration at any time in _Pipeline Settings_.
 
-<%= image "github-settings.png", width: 1700/2, height: 2497/2, alt: "Screenshot of the Buildkite GitHub settings" %>
+## Branch configuration and settings
```

Comment:
We should add one of these to the other repository providers also. Settings are similar but possibly not identical. :-p

Reply:
We don't currently give settings details on the GitHub page. Are you just wanting this note on all pages, or a detailed description of available settings for each, including GH?